pkg/recovery/securerecoverer.go

```diff
@@ -0,0 +1,53 @@
+package recovery
+
+import (
+	"log"
+	"os"
+	"syscall"
+	"time"
+)
+
+const debugTimeout time.Duration = 10
+
+// SecureRecoverer properties
+// Reboot: does a reboot if true
+// Sync: sync file descriptors and devices
+type SecureRecoverer struct {
+	Reboot bool
+	Sync   bool
+	Debug  bool
+}
+
+// Recover by reboot or poweroff without or with sync
+func (sr SecureRecoverer) Recover(message string) error {
+	if sr.Sync {
+		for _, f := range []*os.File{
+			os.Stdout,
+			os.Stderr,
```

Comment:
for two commands it's not really worth generating a list on the fly. Either make this list a top-level public variable (so it can be overridden) or better make it an attribute of the SecureRecoverer. If you do so, you will need to set the struct attribute to a default list (that contains os.Stdout and os.Stderr),so you may need to also implement `func NewSecureRecoverer`.

Reply:
Not needed anymore, I removed the code. Sync should be enough.